File: Array Practice/1512-Number-of-Good-Pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int total = 0;
        for(int i=0; i<nums.length; i++){
            for(int j=0; j<i; j++){
                if(nums[i]==nums[j]) total+=1;
            }
        }
        return total;
    }
}

File: python-practice/1512-Number-of-Good-Pairs.py
class Solution(object):
    def numIdenticalPairs(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        total = 0
        for j in range(len(nums)):
            for i in range(j):
                if nums[i] == nums[j]:
                    total+=1
        return total